test/commands/project/list.test.js

```diff
@@ -18,16 +18,36 @@ describe('commands', () => {
       test
         .do(ctx => {
           ctx.userConfig = new ConfigData();
-          ctx.userConfig.addProject('default', constants.FAKE_ACCOUNT_SID);
+          ctx.userConfig.addProject('project1', constants.FAKE_ACCOUNT_SID);
```

Comment:
Doesn't look like there's a test for when the active project is set (which is causing the coverage to drop).